problem_builder/step.py

```diff
@@ -216,10 +217,18 @@ def _render_view(self, context, view):
             if child is None:  # child should not be None but it can happen due to bugs or permission issues
                 child_contents.append(u"<p>[{}]</p>".format(self._(u"Error: Unable to load child component.")))
             else:
-                child_fragment = self._render_child_fragment(child, context, view)
-
-                fragment.add_frag_resources(child_fragment)
-                child_contents.append(child_fragment.content)
+                if rendering_for_studio and isinstance(child, PlotBlock):
+                    # Don't use view to render plot blocks in Studio.
+                    # This is necessary because:
+                    # - student_view of plot block uses submissions API to retrieve results,
```

Comment:
Oh, ok. I see why you don't have the authoring view showing the plots now. That makes sense. Nevermind, then.